client/client.go

```diff
@@ -395,10 +395,16 @@ type OSRelease struct {
 	VersionID string `json:"version-id,omitempty"`
 }
 
+// RefreshInfo contains information about refreshes.
 type RefreshInfo struct {
-	Schedule string `json:"schedule"`
+	// Schedule contains the legacy refresh.schedule setting. The field is
+	// omitted when the legacy setting is not used.
+	Schedule string `json:"schedule,omitempty"`
```

Comment:
(nitpick^10) I would put schedule and timer close to each other, probably move schedule at the end (under timer).